app/questionPage.html

```diff
@@ -120,6 +135,7 @@ <h4 id="question-counter"></h4>
             </div>
             <div class="col-md-2 right">
                 <h4 class="timer"></h4>
+                <button class="hideTimerButton"></button>
```

Comment:
can you change the name of this class to `showHideTimerButton` or something else that expresses that this is a show button and a hide button (depending on circumstances)